### PR TITLE
Update example.md with new 0.0.9 features

### PR DESCRIPTION
**Pull request by will-hart**
Re #51, updates the `example.md` file with new features.

### CODE REVIEW

File: example.md [hunk 1]
```diff
@@ -162,6 +187,33 @@ Page number is not shown in current page, but it's shown on later pages.
 
 ---
 
+#### Slide background Images
+
+You can set an image background for a slide
```
Comment (yhatt) [inline example.md]: In fact, the background image syntax `bg` has many powerful features like Deckset.

```markdown
## Common use

![bg](images/bg1.jpg)

---

## Disable filter

![bg original](images/bg1.jpg)

---

## Multiple backgrounds

These will align to horizontal.

![bg](images/bg1.jpg)
![bg](images/bg2.jpg)
![bg](images/bg2.jpg)

---

## Sizing

![bg 50%](images/bg1.jpg)
```

In my opinion, I want to include these example for use. On the other hand, example.md would have too many examples..

File: example.md [hunk 2]
```diff
@@ -144,7 +145,31 @@ Example: Set "invert" template of Gaia theme.
 
 ---
 
-#### Pro Tips
+### `footer`
+
+Add a footer to all the presentation slides
```
Comment (yhatt) [inline example.md]: It is not only "to all".

> The page directive would apply to **the current page and the following pages.** You should insert it at the top to apply it to all slides.

File: example.md [hunk 3]
```diff
@@ -162,6 +187,33 @@ Page number is not shown in current page, but it's shown on later pages.
...
 
+#### Slide background Images
+
+You can set an image background for a slide
+
+```html
+![bg](images/marp.png)
+```
+
+![bg](images/marp.png)
```
Comment (yhatt) [inline example.md]: And we would love to set a better background image 😅

**Comment by will-hart** [comment]
Re your comments on the background image, yes maybe the example presentation should be kept relatively small and just give an overview? I wonder if there should be a separate `docs.md` that isn't in presentation form but describes all the features. Then `example.md` can be kept in presentation format and have only the "key" features?

I'll fix up your other comments although I'm not an artist so a better background image could be difficult :)

**Comment by yhatt** [comment]
Nice! I'm with you. Certainly it is very silly to stuff all features to `example.md`.
As you say, Marp have lacked "docs.md" to describe all the features.

It is OK for now but I think we should write `docs.md` until the v0.1.0 minor-release.

File: example.md [hunk 4]
```diff
@@ -162,6 +187,44 @@ Page number is not shown in current page, but it's shown on later pages.
...
+- `original` to include the image without any effects
+- `x%` to include the  image at `x` percent of the slide size
+
+Include multiple`![bg](path)` tags to stack background images horizontally.
+
+![bg original](images/background.png)
+
+---
+
+Multiple background images will be stacked horizontally.
```
Comment (emersion) [inline example.md]: You already said that above.

**Comment by yhatt** [comment]
I'm sorry to have kept you waiting. I'll merge this and prepare v0.0.9 release.

**Pull request merged**